docs/self-hosted/sso.md

```diff
@@ -192,3 +192,17 @@ The configuration disables secured cookies and should only be used in local envi
 your instance to CSRF attacks.
 
 :::
+
+::: tip SSO with Directus-Server running behind a Proxy
+
+If you are running the Directus server behind a HTTP(S)-Proxy to access your SSO provider you need to use global-agent
+for the proxy configuration, e.g.:
+
+```sh
+export GLOBAL_AGENT_HTTP_PROXY=${HTTP_PROXY} &&
+export GLOBAL_AGENT_HTTPS_PROXY=${HTTPS_PROXY} &&
+npx directus bootstrap &&
+node -r 'global-agent/bootstrap' node_modules/directus/dist/start.js
+```
```

Comment:
Let's make sure to update this example to use docker instead of plain npm.

Reply:
Ok, I can adjust that tomorrow :-)

Reply:
Ok, I changed it to Docker  :-)